src/sentry/feedback/blueprints/api.md

```diff
@@ -0,0 +1,336 @@
+| request                    | optional[object] | -                                     |
+| request.headers            | optional[object] | -                                     |
+| request.headers.User-Agent | optional[string] | -                                     |
+| sdk                        | object           | -                                     |
+| sdk.name                   | string           | -                                     |
+| sdk.version                | string           | -                                     |
+| tags                       | optional[object] | -                                     |
+| timestamp                  | number           | UTC timestamp.                        |
+| user                       | optional[object] | -                                     |
+| user.email                 | optional[string] | -                                     |
+| user.id                    | optional[string] | -                                     |
+| user.ip_address            | optional[string] | -                                     |
+| user.name                  | optional[string] | -                                     |
+| user.username              | optional[string] | -                                     |
+| contexts                   | optional[object] | -                                     |
+| BrowserContext             | optional[object] | -                                     |
+| name                       | optional[string] | -                                     |
+| version                    | optional[string] | -                                     |
+| DeviceContext              | optional[object] | -                                     |
+| name                       | optional[string] | -                                     |
+| model                      | optional[string] | -                                     |
+| type                       | optional[string] | -                                     |
+| family                     | optional[string] | -                                     |
+
+- Request
+
+  ```json
+  {
+    "dist": "abc123",
+    "environment": "production",
+    "event_id": "1ffe0775ac0f4417aed9de36d9f6f8dc",
+    "feedback": {
+      "contact_email": "colton.allen@sentry.io",
+      "message": "I really like this feedback feature!",
+      "replay_id": "ec3b4dc8b79f417596f7a1aa4fcca5d2",
+      "url": "https://docs.sentry.io/platforms/javascript/"
+    },
+    "platform": "javascript",
+    "release": "version@1.3",
+    "request": {
+      "headers": {
+        "User-Agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/103.0.0.0 Safari/537.36"
+      }
+    },
+    "sdk": {
+      "name": "sentry.javascript.react",
+      "version": "6.18.1"
+    },
+    "tags": {
+      "key": "value"
+    },
+    "timestamp": 146990235.23,
+    "user": {
+      "email": "username@example.com",
+      "id": "123",
```

Comment:
could we standardize whether we're calling it `feedback_id` (in the path) or `id` (here)?

Reply:
also, I believe in the ingestion tests we take in `event_id` rather than just `id`

the pipeline currently is:
POST takes in `event_id` --> STORE IN DB as `feedback_id` --> SERIALIZE as `id` and returned to user as `id`

cc @JoshFerge

Reply:
@michellewzhang the id on the line you commented is referring to the user id rather than the feedback_id.

Reply:
oops let me move my comment